Inc/stm32f446xx_gpio.h

```diff
@@ -10,27 +10,51 @@
 
 #include<stdint.h>
 #include"stm32f446xx.h"
+#include <iostream>
```

Comment:
no need here. iostream works only in full fledged g++ environ. uC uses arm-g++ compiler, making iostream irrelevant.

Reply:
ok